chain-impl-mockchain/src/ledger.rs

```diff
@@ -156,7 +156,7 @@ impl Ledger {
                 params.block0_initial_hash = block0_hash;
                 Ok(params)
             }
-            Some(_) => Err(Error::Block0InitialMessageMissing),
+            _ => Err(Error::Block0InitialMessageMissing),
```

Comment:
no, the error message is suppose to be different in the first place (I used the already existing error when adding this big, and forgot to add the new error). in the case of `Some(_)` it should be a `InitialMessageExpectedButGotSomethingElse`